content/guides/accessibility-for-teams/ux-design.md

```diff
@@ -0,0 +1,176 @@
+- [GDS’s guidance on form structure](https://www.gov.uk/service-manual/design/form-structure)
+- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG20/quickref/#text-equiv-all)
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG20/quickref/#content-structure-separation-programmatic)
+- [3.2.1 On Focus](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=128%2C14&currentsidebar=%23col_overview#consistent-behavior-receive-focus)
+- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG20/quickref/#minimize-error-cues)
+- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG20/quickref/#ensure-compat-rsv)
+
+## Images
+{{< ring title="Why it's important" >}}
+Have you identified which images are meaningful and which images are decorative in mockups?
+
+- Carmen’s page didn’t load all the way and didn’t get to download the images. 
+- Amanda is blind and uses a braille reader to understand the content of images. 
+- John is looking for information with a search engine and needs help being directed to the right content (descriptive alt tags will improve search).
+{{< /ring >}}
+
+### Steps to take
+- Images that are purely decorative shouldn’t be announced by the screenreader, work with your front-end developer to make sure they’re coded correctly.
+- Images that have meaning should include alt text and possibly a longer description, work with the content designer on your team to create these.
+- Making sure any text in images of text is at least 14 points and has good contrast.
+
+### Resources
+- [1.1 Text Alternatives (Guideline)](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=11%2C111#text-equiv)
+- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=14%2C128&currentsidebar=%23col_overview#text-equiv-all)
+- [1.4.5 Images of Text](https://www.w3.org/WAI/WCAG20/quickref/#qr-visual-audio-contrast-text-presentation)
+
+## Touch targets
+{{< ring title="Why it's important" >}}
+Are your touch targets large enough and easy to reach?
+
+- Harold has gigantism due to a tumor on his pituitary gland during childhood. He has very large hands which makes small links, or links that are too close together difficult to tap on his phone. 
+- Rosa checks the latest news on her phone during her morning commute on a jostling bus, often while sipping from the coffee cup in her other hand.
+{{< /ring >}}
+
+### Steps to take
+- Make sure you can reach primary actions easily with either right or left thumbs, even on larger phones. Items on the bottom of the screen tend to be easier to reach.
+- Make touch targets at least 44 px or 10 mm. This will allow the target to be tapped by the average adult finger pad, which measures 10 mm. The icons may be smaller and you can work with your developer to extend padding past the icon. Pay particular attention to how [line-height affects touch target size](http://www.3needs.org/en/testing/target-size.html).
```

Comment:
```suggestion
- Make touch targets at least 44 px or 10 mm. This will allow the target to be tapped by the average adult finger pad, which measures 10 mm. The icons may be smaller and you can work with your developer to extend padding past the icon. Pay particular attention to how [line-height affects touch target size](https://digital.gov/guides/mobile-principles/tap-targets/).
```